composer.json

```diff
@@ -17,13 +17,13 @@
         "j13k/yaml-lint": "^1.1",
         "sserbin/twig-linter": "^3.0",
         "mockery/mockery": "^1.4",
-        "behat/mink": "^1.8",
-        "symfony/phpunit-bridge": "^5.3",
+        "behat/mink": "^1.10",
+        "symfony/phpunit-bridge": "^6.1",
```

Comment:
Do we also need to support the ^5 version too? What's the constraint if any in Drupal core?